src/stars.css

```diff
@@ -0,0 +1,37 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  background-image: url('images/star.svg');
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 12px;
+  width: 13px;
+  margin: 2px 5px 2px 2px;
```

Comment:
it would be better to just follow design parameters :) 
change height and width to 16px and margin only to margin right 4px. 
Additionally you need to center the image so delete `background-size: cover` to keep original size and add `background-position: center` to center it 
![image](https://github.com/user-attachments/assets/42f28ab8-6ef7-4edd-a9a3-c73183555c46)

Reply:
I thought that I had to set the size of the stars to the same size as in the svg file, i.e. 13/12, because when I set the size to 16/16 it cut off a piece of the star arm, probably due to the lack of "background position".. now it works correctly!